src/main/res/values/strings.xml

```diff
@@ -160,38 +160,38 @@
     <string name="unstable_updates">Unstable updates</string>
     <string name="unstable_updates_summary">Suggest installing unstable versions</string>
     <string name="unverified">Unverified</string>
-    <string name="update">Update</string>
-    <string name="updates">Updates</string>
-    <string name="upstream_source_code_is_not_free">The upstream source code is not free</string>
+    <string name="update">Upgrade</string>
+    <string name="updates">Upgrades</string>
```

Comment:
```suggestion
    <string name="updates">Apps with new versions</string>
```

Reply:
I guess you're mixing my feedback fully and possibly didn't take a look in the app (at all). So here some main lines:
- Update is the verb update and it's explicitly associated with updating an app (downloading and installing the new version).
- Updates is a list of the apps with new versions but its far more adequate than using the full description.
- Most of the action texts e.g. update all should be as compact as it can, considering the limited button space. The same applies to the titles e.g. updates.
- This is the main language and not a simple variant, so it doesn't need to be using the simplest terms.

So please navigate through the app extensively (this would help in better understanding the context - if have not done till now) and consider my previous notes. Till now I have the feeling that we're just talking past each other.